scripts/04233-femathic.js

```diff
@@ -1,4 +1,5 @@
 const fullName = 'Oluwafemi Opanuga';
 const hngId = "HNG-04233";
+const email = "femathic@gmail.com"
 
-console.log(`Hello World, this is ${fullName} with HNGi7 ID ${hngId} using JavaScript for stage 2 task`);
+console.log(`Hello World, this is ${fullName} with HNGi7 ID ${hngId} and email ${email} using JavaScript for stage 2 task`);
```

Comment:
Hi, you need to move the email to the end of the sentence as highlighted [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

Reply:
Thanks Ifatoki, I just did that.